src/main/java/duke/command/AddCommand.java

```diff
@@ -0,0 +1,67 @@
+package duke.command;
+
+import duke.exception.DukeException;
+import duke.io.Ui;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.TaskList;
+import duke.task.Todo;
+
+public class AddCommand extends Command {
```

Comment:
Using inheritance here is a good idea! An alternate suggestion would be to have an enum for commands.